src/main/views/status-description.njk

```diff
@@ -0,0 +1,71 @@
+{% from "govuk/components/table/macro.njk" import govukTable as 'govukTable' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{{ displayGlossaryAlphabet }}
+{% extends "template.njk" %}
+{% block pageTitle %}
+  {{ "Event Glossary" | safe }}
+{% endblock %}
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: text,
+    href: '#',
+    attributes: {
+      onClick: "history.back();"
+    }
+  }) }}
+{% endblock %}
+{% block content %}
+  <div id="grid-row" class="govuk-grid-row">
+    <div class="layout-width-four-fifths">
+      <h2 id='page-heading' class="govuk-heading-l">Live hearing updates - glossary of terms</h2>
+      <p class="govuk-body">A list of descriptions for each hearing status that reflects in-court events</p>
+      <div class="govuk-grid-row govuk-!-margin-bottom-8">
+        {% for key, value in statusList %}
+          {% if value| length != 0 %}
+            <div class="govuk-grid-column width-column left-align-text govuk-!-font-size-19">
+              <a id='{{ key }}-selector' href='#{{ key }}' class="govuk-link govuk-link--no-visited-state">{{ key }}</a>
+            </div>
+          {% endif %}
+        {% endfor %}
+      </div>
+
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header app-custom-class"></th>
+          <th scope="col" class="govuk-table__header govuk-!-width-one-quarter">Hearing status</th>
+          <th scope="col" class="govuk-table__header govuk-!-width-three-quarters">Description</th>
+        </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+        {% for key, value in statusList %}
+          {% for status, statusDetails in value %}
+            <tr class="govuk-table__row">
+              {% if value | length != 0 %}
+                  <th scope="row" class="govuk-table__header">
+                  {% if displayGlossaryAlphabet != "false" %}
+                      <h2 id="{{ key }}" class="govuk-heading-l">{{key}}</h2>
+                  {% endif %}
+                  </th>
+                <td class="govuk-table__cell">
+                  <div id='{{ statusDetails.id }}' class="govuk-grid-column">
+                    <a id='{{ statusDetails.id }}-selector' href='#{{ statusDetails.id }}' class="link-without-formatting">{{ status }}</a>
```

Comment:
why are these links?

Reply:
As per ticket, these status will be linked to other page. When user clicks on the link, it will take user to this page and direct to that status.

Reply:
the link is on the page before this though, im talking about the hrefs you have linking to themselves

Reply:
No need to use anchor tag. using div and also removed the custom class.